Codegen/test/resources/input/TypeCobol/CopyReplace4Colon/CopyReplace4Colon5.rdz.cbl

```diff
@@ -0,0 +1,13 @@
+﻿       IDENTIFICATION DIVISION.
+       PROGRAM-ID. CPYRPL4C.
+      
+       ENVIRONMENT DIVISION.
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+       01  xxxENT. COPY YxxxENT REPLACING ==::== BY ====.
+       01  xxxENT2. COPY YxxxENT2 REPLACING ==::== BY ====.
+       PROCEDURE DIVISION.
+           MOVE 'A' TO xxxENT-FCT01-Var1 OF xxxENT.
+           MOVE 'A' TO xxxENT2-FCT01-Var1.
```

Comment:
These lines:
```cobol
01  xxxENT2. COPY YxxxENT2 REPLACING ==::== BY ====.
MOVE 'A' TO xxxENT2-FCT01-Var1.
```
could be merged into test CopyReplace4Colon3.rdz.cbl, so this test 5 can be removed.

Reply:
This test is not only intended to test MOVE 'A' TO xxxENT2-FCT01-Var1.
But also to test COPY inclusion sequence.